com.unity.ml-agents/Runtime/Timer.cs

```diff
@@ -28,12 +28,6 @@ internal class TimerNode
         [DataMember(Name = "children", Order = 999)]
         Dictionary<string, TimerNode> m_Children;
 
-        /// <summary>
-        /// Gauge Nodes to measure arbitrary values.
-        /// </summary>
-        [DataMember(Name = "gauges", EmitDefaultValue = false)]
```

Comment:
Since only the root node needs gauges and metadata, I made the RootNode a separate class. Just a memory micro-optimization.